steps/amazon-s3-uploader/1.0/step.yml

```diff
@@ -0,0 +1,105 @@
+title: Amazon S3 File uploader
+summary: Upload a file to Amazon S3
+description: |-
+  Bitrise Step to upload a file to Amazon S3.
+
+  This Step requires an Amazon S3 account.
+  To register an Amazon S3 account, [click here](http://aws.amazon.com/s3/).
+website: https://github.com/scruffyfox/bitrise-amazon-s3-uploader
+source_code_url: https://github.com/scruffyfox/bitrise-amazon-s3-uploader
+support_url: https://github.com/scruffyfox/bitrise-amazon-s3-uploader/issues
+published_at: 2016-09-16T15:29:59.994124301+01:00
+source:
+  git: https://github.com/scruffyfox/bitrise-amazon-s3-uploader.git
+  commit: 1538d8a978516687b098c52e2a7812ca9424cf68
+type_tags:
+- deploy
+- amazon
+- s3
+- upload
+- uploader
+deps:
+  brew:
+  - name: awscli
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: true
```

Comment:
Yes, i left it like this because in our use case, its not always required for the upload to happen, if it doesnt it shouldn't fail the build, but I can change this if it makes more sense?